presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -382,3 +382,119 @@ node.
 
 You should ensure that the keytab files have the correct permissions on every
 node after distributing them.
+
+
+Hive Data Stored in Amazon S3
+=============================
+
+The Hive Connector can read data from files stored in S3, assuming the Hive
+Metastore has tables defined as "external" which have locations defined as
+S3 URIs.
+
+Presto registers its own S3 filesystem for the following URI schemes:
+``s3://``, ``s3a://``, ``s3n://`` and it registers the default Hadoop S3
+FileSystem (``org.apache.hadoop.fs.s3.S3FileSystem``) for ``s3bfs://`` URIs.
+The following assumes your are using the native Presto S3 FileSystem implementation.
+
+Configuration Properties
+------------------------
+
+============================================ ====================================================== ==========
+Property Name                                Description                                            Default
+============================================ ====================================================== ==========
+``presto.s3.access-key``                     Default API access key to use
```

Comment:
Ah, thanks -- I always forget about that translation layer from HiveClientConfig.

However, for "presto.s3.encryption-materials-provider" this is contained entirely in PrestoS3FileSystem - should the same be true for presto.s3.kms-key-id ?